data/human/human missions.txt

```diff
@@ -5416,3 +5416,105 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+mission "Quicksilver Mixup [0]"
+	name "Package to Silver"
+	description "Deliver a package weighing <tons> to Silver by <date>. Payment is <payment>."
+	"apparent payment" 17100
+	cargo package 1
+	deadline 3 0
+	to offer
+		random < 25
+	source
+		near Algol 3
+	destination Silver
+	on offer
+		conversation
+			`You're looking at some job listings when a man with a harried face approaches you, carrying a package.`
+			`	"Excuse me, Captain," he says. "I need this package delivered in the next three days. Details are on the package. I'll pay you <payment> once it's done." Without waiting for a response, he shoves the package into your hands and runs off to attend to whatever business has gotten him in such a hurry. The package is addressed to a location on Silver.`
+			branch "can't make it in time"
+				not "hyperjumps to system: 3"
+			`	It's going to be close to his deadline, but it's possible.`
+				goto choice
+			label "can't make it in time"
+			`	How he expects you to deliver that package in three days is beyond you, seeing as Silver is not within three jumps of <origin>, but you might as well try and get there as soon as possible.`
+			label choice
+			choice
+				`	(Deliver the package.)`
+				`	(Find another pilot to give the package to.)`
+					goto declined
+			`	You take the package to your ship, leave it in the cargo bay, and prepare to fly to Silver.`
+				accept
+			
+			label declined
+			`	It takes a while searching for a pilot who can handle the delivery, but you do eventually find one. You hand them the package, explain the situation, thank them, and leave.`
+				decline
+	on visit
+		dialog phrase "generic cargo on visit"
+
+mission "Quicksilver Mixup [1]"
+	name "Redirect Package to Quicksilver"
+	description "Apparently the package of <tons> labeled for Silver was actually intended for Quicksilver. You'll have missed the deadline, so you probably won't be paid the <payment> offered."
+	landing
+	cargo package 1
+	"apparent payment" 12100
```

Comment:
```suggestion
	"apparent payment" 17100
```
It was 17100 before, and with the bonus 5k it is 22100, so I guess this one is wrong.